storybook/webpack.config.js

```diff
@@ -1,30 +1,33 @@
-const findCacheDir = require('find-cache-dir');
-
 module.exports = {
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.(?:png|svg)$/,
-        loader: 'url-loader',
+        use: 'url-loader',
       },
       {
         test: /\.jsx?$/,
         exclude: [/(node_modules)/, /react-css-themr/],
-        loader: require.resolve('babel-loader'),
-        query: {
-          babelrc: false,
-          cacheDirectory: findCacheDir({ name: 'react-storybook' }),
-          presets: ['env', 'react'],
-          plugins: [
-            'transform-flow-strip-types',
-            'transform-decorators-legacy',
-            'transform-class-properties',
-            'lodash',
-          ],
-        },
-      }, {
+        use: [{
+          loader: 'babel-loader',
+          options: {
+            babelrc: false,
+            presets: [
+              ['env', { "modules": false }],
+              'react'
+            ],
+            plugins: [
+              'transform-flow-strip-types',
+              'transform-decorators-legacy',
+              'transform-class-properties',
+              'lodash',
+            ],
+          }
+        }],
+      },
+      {
         test: /\.scss$/,
-        loaders: [
+        use: [
           'style-loader?sourceMap',
           'css-loader?sourceMap&modules&localIdentName=[name]_[local]&importLoaders=1',
```

Comment:
`options: {}` in new webpack syntax look nicer.

Reply:
Good point, is updated! 👍